viewer/packages/360-images/src/annotation/ImageAnnotationObject.ts

```diff
@@ -20,12 +32,15 @@ import { Image360AnnotationAppearance } from './types';
 type FaceType = Image360FileDescriptor['face'];
 
 import SeededRandom from 'random-seed';
+import { ThickLine } from '@reveal/utilities';
 
 export class ImageAnnotationObject implements Image360Annotation {
   private readonly _annotation: AnnotationModel;
 
   private readonly _mesh: Mesh;
-  private readonly _material: MeshBasicMaterial;
+  private readonly _meshMaterial: MeshBasicMaterial;
+  private readonly _lines: ThickLine[];
+  private readonly _objectGroup: Group;
```

Comment:
Not sure what is better but in case of `OverlayPointsObject` I was extending the whole class to be `Object3D` instead of having a group/object3d property within the calls. Maybe it is also worth considering here? But up to you to decide.

Reply:
I don't think it matters that much.. But I decided to do composition and not inheritance when I started working on annotations, so I'll keep going with that, I think